code-highlighter/code-highlighter.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for Code-Highlighter v2.2.0
+// Project: https://github.com/avielfedida/Code-Highlighter
+// Definitions by: Aviel Fedida <https://github.com/avielfedida/>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+interface IHighlighter {
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.